src/monocle_apptrace/wrap_common.py

```diff
@@ -427,6 +403,62 @@ def update_span_from_llm_response(response, span: Span, instance):
             token_usage = None
 
 
+def extract_messages(args):
+    """Extract system and user messages"""
+    system_message, user_message = "", ""
+    args_input = get_attribute(DATA_INPUT_KEY)
```

Comment:
Please add try except in all these methods to catch and log any errors

Reply:
done